sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -38,9 +38,11 @@
     <packageOutputDirectory>${project.build.directory}</packageOutputDirectory>
     <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
     <azure.spring.boot.version>${project.version}</azure.spring.boot.version>
+    <azure.spring.boot.patch.version>3.6.1</azure.spring.boot.patch.version>
```

Comment:
Do not use patch version here, and azure.spring.boot.version will resolve to 3.6.1. If packages are not released this time, just list their versions in property.